ddtrace/tracer/context.go

```diff
@@ -9,30 +9,27 @@ import (
 	"context"
 
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/internal"
+	tracerInternal "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/internal"
```

Comment:
```suggestion
	traceinternal "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/internal"
```

let's use the same alias as we are using in other files (https://github.com/DataDog/dd-trace-go/blob/main/ddtrace/tracer/transport.go#L22)